_deployment/minify-js.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/sh
PATH="./node_modules/.bin:$PATH"
```
Now this should also work for people that have installed the node modules locally.